### PR TITLE
Add map with markers

### PR DESCRIPTION
**Pull request by andreram**
Changes:
* Add a map to the itinerary page with markers that you can add on click.
* Multiple markers can be added, and a marker can be removed by clicking on it again (this function will probably change to display the info of the associated timeslot later on).
* Fixed some console warnings

### CODE REVIEW

File: trippo/src/components/map/Map.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.mapboxgl-marker {
```
Comment (JoshLuong) [inline trippo/src/components/map/Map.css:1]: Is it possible to use styled components so we remain consistent across the app?

File: trippo/src/components/map/Map.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import "./Map.css";
+
+interface State {
+  lng: number;
+  lat: number;
+  zoom: number;
+  // marker: mapboxgl.Marker[];
+}
+
+export default class Map extends React.Component<{}, State> {
```
Comment (JoshLuong) [inline trippo/src/components/map/Map.tsx]: Is it possible to use functional components/ hooks so we're more consistent?

File: trippo/src/components/map/Map.css (hunk 1 above)
Comment (andreram) [reply trippo/src/components/map/Map.css:1]: This one is a little weird for styled-components. Working with the mapbox library while trying to keep things like normal React and not pure JS (think document.getElementsByClassName) is tough. I'd generally prefer consistency but in this case I thought the trouble was just not worth it.

I might look into some React-friendly packages if I keep running into problems, but at the moment I think we should focus on fleshing out our MVP.

File: trippo/src/components/map/Map.tsx (hunk 2 above)
Comment (andreram) [reply trippo/src/components/map/Map.tsx]: Done

**Review by r-bassi — APPROVED** [review]

**Pull request merged**